Cloudinary/CLDConfiguration.swift

```diff
@@ -101,13 +101,17 @@ import Foundation
      */
     public static func initWithEnvParams() -> CLDConfiguration? {
         let dict = ProcessInfo.processInfo.environment
-        if let url = dict[Defines.ENV_VAR_CLOUDINARY_URL] {
+        if let url = dict[Defines.ENV_VAR_CLOUDINARY_URL], validateUrl(url: url) == true {
```

Comment:
I believe you don't need the `== true` at the end, it was put there since validate returned an optional boolean at some point if I remember correctly.